src/bin/flamegraph.rs

```diff
@@ -31,15 +32,61 @@ struct Opt {
         )
     )]
     colors: Palette,
+
     /// set background colors. Gradient choices are yellow (default), blue, green, grey; flat colors use "#rrggbb"
     #[structopt(long = "bgcolors")]
     bgcolors: Option<BackgroundColor>,
+
     /// colors are keyed by function name hash
     #[structopt(long = "hash")]
     hash: bool,
+
     /// use consistent palette (palette.map)
     #[structopt(long = "cp")]
     cp: bool,
+
+    /// Change title text
+    #[structopt(name = "title", long = "title")]
```

Comment:
Is it necessary to give a `name` to all of these?

Reply:
made a cleanup of names